common/services/healthcheck.js

```diff
@@ -1,15 +1,15 @@
 const { serviceCheckFactory } = require('../data/healthcheck')
 
-const service = (name, url) => {
-  const check = serviceCheckFactory(name, url)
+const service = (name, config) => {
+  const check = serviceCheckFactory(name, config)
   return () =>
     check()
       .then(result => ({ name, status: 'ok', message: result }))
       .catch(err => ({ name, status: 'ERROR', message: err }))
 }
 
-module.exports = (auth, sentencePlanning, elite2) => {
-  const checks = [service('auth', auth), service('sentencePlanning', sentencePlanning), service('elite2', elite2)]
+module.exports = (...services) => {
+  const checks = services.map(s => service(s.name, s.config))
```

Comment:
de-struct in the arguments:
const checks = services.map(({name, config}) => service(name, config))

Reply:
I much prefer the services object though 👍

Reply:
done